src/main/webapp/help/workingDir.html

```diff
@@ -0,0 +1,2 @@
+Path to the root of the workspace from the view point of this node, such as <em>/home/jenkins/agent</em>.<br/>
+This doesn't need to be absolute since the launcher establishes a consistent working directory, such as <em>./.jenkins-agent</em>.
```

Comment:
I am not sure what this second sentence is about. I think it is copied from some core help? Probably better to delete it; users should not rely on the CWD of the init process.